service/internal/hamaster/ha_master.go

```diff
@@ -53,7 +53,7 @@ func (h *HAMaster) Enabled(ctx context.Context, cluster string) (bool, error) {
 	err := h.k8sClient.CtrlClient().List(
 		ctx,
 		&list,
-		client.InNamespace(v1.NamespaceDefault),
+		client.InNamespace(v1.NamespaceAll),
```

Comment:
I think we can also just drop the line in this case, maybe?

Reply:
Yeah AFAIK `NamespaceAll` is just "". It might be tidier to drop it and more specific to keep it. No preference from me.